com.github.petebuffon.yafc.yml

```
@@ -0,0 +1,30 @@
+app-id: com.github.petebuffon.yafc
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: YAFC
+finish-args:
+  - "--device=dri"
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--filesystem=~/.factorio/mods:ro"
+  - "--filesystem=~/.steam/steam/steamapps/common/Factorio/data:ro"
+  - "--filesystem=xdg-run/app/com.valvesoftware.Steam:ro"
```

You probably intended to access Steam flatpak data, not runtime dir.
```suggestion
  - "--filesystem=~/.var/app/com.valvesoftware.Steam:ro"
```

---

Great, commit added.